src/dfx/src/commands/replica.rs

```diff
@@ -63,27 +77,59 @@ fn get_port(config: &ConfigDefaultsReplica, port: Option<String>) -> DfxResult<u
         .map_err(|err| error_invalid_argument!("Invalid port number: {}", err))
 }
 
-fn start_replica(
-    env: &dyn Environment,
-    opts: ReplicaOpts,
-    shutdown_controller: Addr<ShutdownController>,
-) -> DfxResult {
-    let replica_config = get_config(env, opts)?;
-    start_replica_actor(env, replica_config, shutdown_controller)?;
-    Ok(())
-}
-
 /// Start the Internet Computer locally. Spawns a proxy to forward and
 /// manage browser requests. Responsible for running the network (one
 /// replica at the moment) and the proxy.
```

Comment:
```suggestion
/// replica at the moment), the proxy, and (if configured) the bitcoin adapter.
```
AFAIU the adapter only gets started if it is configured either through CLI option or in dfx.json. Is it possible to make that a little more explicit? Right now that condition is hidden in line 113, where you may notice it or not. An `if let Some` would make that much more readable for me personally.

Reply:
Great point.  Thanks!